lib/srv/regular/sshserver.go

```diff
@@ -623,7 +623,7 @@ func (s *Server) AdvertiseAddr() string {
 	if aport == "" {
 		aport = port
 	}
-	return net.JoinHostPort(ahost, port)
```

Comment:
Can you add a test, that will also help clear up expected behavior here.

Reply:
Done, also renamed `advertiseIP` to `advertiseAddr` to better reflect what it is.